src/ui/public/visualize/visualize.js

```diff
@@ -81,22 +68,18 @@ uiModules
 
         $scope.vis.description = $scope.savedObj.description;
 
-        $scope.editorMode = $scope.editorMode || false;
-        $scope.vis.editorMode = $scope.editorMode;
```

Comment:
By removing this, we'll also need to remove the debug output inside Vega, that relies on this being set.

Reply:
Maybe Vega could rely on `vis.isEditorMode()` method since in visualize we are not using anymore the editorMode variable, thus it will be always false, as the default return from isEditorMode().

Reply:
I think the vis itself should not have any knowledge about it running in editor mode anymore. We should also remove `vis.isEditorMode()`. Especially since the assumption here is that as long as we're running in editor mode we're the only vis rendered. Hopefully we'll have better in-place editing some time so we can edit on the dashboard directly which would render that assumption wrong.

I also warned Yuri that we'll remove that flag rather soon when it was built into Vega and that we'll remove that debugging output in that case, so we are stilling following the outlined plan :)